.env.development

```diff
@@ -1,4 +1,4 @@
-REACT_APP_NEW_TEACHER_SITE=false
+REACT_APP_NEW_TEACHER_SITE=true
```

Comment:
till it's completely developed; this flag should only be enabled on your branch.

Reply:
Added the .env.development file to .gitignore